CONTRIBUTING.md

```diff
@@ -2,31 +2,41 @@
 
 Jest to drugi najważniejszy plik po README w projekcie. Dotyczy on osób, które chcą w sposób aktywny wpłynąć na rozwój naszego forum.
 
-# Tworzenie `issue`
+## Standardy kodu
+
+Tworzony kod PHP **musi przestrzegać standardu [PSR-2](http://www.php-fig.org/psr/psr-2/)** tam, gdzie tylko jest to możliwe - zdajemy sobie sprawę, że w niektórych miejscach będzie to niewykonalne przez już napisany kod Question2Answer. Twój kod musi działać z **PHP w wersji 5.6** - z takiej obecnie korzystamy.
```

Comment:
musi przestrzegać standardów PSR-1 oraz PSR-2 tam, gdzie tylko jest to możliwe. 
Przy okazji warto wspomnieć o zasadzie dobrego skauta. Jeśli tylko mamy czas i chęci to będąc w istniejącym już pliku staramy się samodzielnie poprawić na przykład formatowanie czy część kodu by był czytelniejszy. Poprawa formatowania może spowodować bardzo nieczytelne commity (nie będzie wiadomo, co zostało zmienione). Z tego powodu warto najpierw - przed rozpoczęciem pracy poprawić formatowanie, puścić commita i dopiero po tym zacząć pracę nad swoim. Lub w kolejności odwrotnej - ważne, że jeśli się ktoś na to zdecyduje, to niech rozdzieli poprawki formatowania od samych zmian w kodzie.

Reply:
Czemu nie, można jakoś krótko o tym wspomnieć